content/automate/ManageIQ/System/Event/EmsEvent/Kubernetes.class/pod_containercreated.yaml

```diff
@@ -4,9 +4,11 @@ version: 1.0
 object:
   attributes:
     display_name: 
-    name: CONTAINER_FAILED
+    name: POD_CONTAINERCREATED
```

Comment:
I can't find where this event is created?

Reply:
Looks like a weird git error. 
container_created and container_failed got criss-crossed in file change rename

`Kubernetes.class/container_created.yaml → ...Kubernetes.class/pod_containerfailed.yaml`
`Kubernetes.class/container_failed.yaml     → ...Kubernetes.class/pod_containercreated.yaml`

Reply:
OK didn't found where the POD prefix is being added. @zeari [located it]( https://github.com/ManageIQ/manageiq-providers-kubernetes/blob/3345798e7b1ac541e200beac99e546ffafc606ce/app/models/manageiq/providers/kubernetes/container_manager/event_catcher_mixin.rb#L107)